Request for Code Review: Building with Backgrounds & Gradients

edieatha opened the pull request:
## Apple Page Clone:
1. Used background image for header image.
2. Applied a gradient on navigation buttons.
3. Used positioning 'absolute' on all sections of the page.
4. Aligned footer and aside elements with flex.
5. Added images and text styles.

mikenath223 commented:
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 👋

mikenath223 commented on index.html:
```diff
@@ -0,0 +1,114 @@
+        </nav>
+
+			<div class="top-container">
+				<div class="heading"><a href="#"><img class="small"
+						src="images/title_small.png"
+						alt="What will your verse be?" width="980" height="90"></a></div>
+				<div class="summary"><a href="#"><img class="small"
+						src="images/subhead_small.png" alt="Everyone has something to share."
+			                          width="980" height="82"></a></div>
+				<div class="watch">
+					<a href="#"><img class="small"
+						src="images/cta_your_verse_01.png" alt="Your verse"
+						 width="980" height="34"></a> </div>
+				</div>
+
```
Convert these image elements to text elements instead.

mikenath223 commented on index.html:
```diff
@@ -0,0 +1,114 @@
+
+	</main>
+	<aside class="bottom-container">
+		<div>
+			<a href="#"><img src="images/iphone5s.jpg"
+					 alt="iPhone 5s"></a>
+		</div>
+		<div>
+	               <a href="#"><img src="images/iphone5c.jpg"
+					   alt="iPhone 5c"></a>
+		</div>
+		<div>
+			<a href="#"><img src="images/promo_30_years.jpg"
+					 alt="Happy Birthday Apple"></a>
+		</div>
+		<div>
+			<a href="#"><img src="images/supplier_responsibility.jpg"
+					 alt="Supplier Responsibility"></a>
+		</div>
```
Convert the images here also to text elements

mikenath223 commented on index.html:
```diff
@@ -0,0 +1,114 @@
+            <li class="gn-support"><a href="#"><span>Support</span></a></li>
+          </ul>
+          <div class="globalsearch">
+            <form class="g-search">
+                <div class="magnify">
+                    <img src="images/search-icon.png" alt="search" height="13" width="13">
+                </div>
+            </form>
```
Kindly change this into a search input field. Also fix the HTML validation errors in the HTML code with this [tool](https://validator.w3.org/nu/#file).

edieatha commented:
## TSE Code Review Changes

1. Converted image elements in HTML to background CSS elements.
2. Changed search element to input type element
3. Changed text images to actual text elements
4. Validated HTML
